addon/components/polaris-range-slider.js

```diff
@@ -0,0 +1,289 @@
+   */
+  labelHidden: false,
+
+  /**
+   * ID for range input
+   *
+   * Defaults to Ember's internal GUID for the component instance
+   *
+   * @property {id}
+   * @type {String}
+   * @public
+   */
+  id: computed(function() {
+    return guidFor(this);
+  }),
+
+  /**
+   * Initial value for range input
+   *
+   * @property {value}
+   * @type {Number}
+   * @default 0
+   * @public
+   */
+  value: 0,
+
+  /**
+   * Minimum possible value for range input
+   *
+   * @property {min}
+   * @type {Number}
+   * @default 0
+   * @public
+   */
+  min: 0,
+
+  /**
+   * Maximum possible value for range input
+   *
+   * @property {max}
+   * @type {Number}
+   * @default 100
+   * @public
+   */
+  max: 100,
+
+  /**
+   * Increment value for range input changes
+   *
+   * @property {step}
+   * @type {Number?}
+   * @default null
+   * @public
+   */
+  step: null,
+
+  /**
+   * Provide a tooltip while sliding, indicating the current value
+   *
+   * @property {output}
+   * @type {Boolean}
+   * @default false
+   * @public
+   */
+  output: false,
+
+  /**
+   * Additional text to aid in use
+   *
+   * @property {helpText}
+   * @type {String|Component}
+   * @public
+   */
+  helpText: null,
+
+  /**
+   * Display an error message
+   *
+   * @property {error}
+   * @type {String|Component}
+   * @public
+   */
+  error: null,
+
+  /**
+   * Disable input
+   *
+   * @property {disabled}
+   * @type {Boolean}
+   * @default false
+   * @public
+   */
+  disabled: false,
+
+   /**
+   * Callback when the range input is changed
+   *
+   * @property {onChange}
+   * @type {Function}
+   * @default no-op
+   * @public
+   */
+  onChange() {},
+
+  /**
+   * Callback when range input is focused
+   *
+   * @property {onFocus}
+   * @type {Function}
+   * @default no-op
+   * @public
+   */
+  onFocus() {},
+
+  /**
+   * Callback when focus is removed
+   *
+   * @property {onBlur}
+   * @type {Function}
+   * @default no-op
+   * @public
+   */
+  onBlur() {},
+
+  /**
+   * Class names for the range input wrapper div
+   *
+   * @property {rangeInputWrapperClassNames}
+   * @type {String}
+   * @private
+   */
+  rangeInputWrapperClassNames: computed('error', 'disabled', function() {
+    let { error, disabled } = this.getProperties('error', 'disabled');
+    let classNames = ['Polaris-RangeSlider'];
+
+    if (error) {
+      classNames.push('Polaris-RangeSlider--error');
+    }
+
+    if (disabled) {
+      classNames.push('Polaris-RangeSlider--disabled');
+    }
+
+    return classNames.join(' ');
+  }).readOnly(),
+
+  /**
+   * Style for the range input wrapper div
+   *
+   * @property {rangeInputWrapperStyle}
+   * @type {String}
+   * @private
+   */
+  rangeInputWrapperStyle: computed('min', 'max', 'value', 'sliderProgress', function() {
+    let { min, max, value: current, sliderProgress } = this.getProperties('min', 'max', 'value', 'sliderProgress');
+    let styleProps = assign({ min, max, current }, {
+      current,
```

Comment:
is this `current` needed here, since it's also in the object in which it's merged?

Reply:
Eagle-eyes @vladucu 😅 Thanks, this was left over from a previous iteration 👍